test/e2e/scheduling/predicates.go

```diff
@@ -485,12 +488,7 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 			NodeSelector: map[string]string{labelKey: labelValue},
 		})
 
-		// check that pod got scheduled. We intentionally DO NOT check that the
-		// pod is running because this will create a race condition with the
-		// kubelet and the scheduler: the scheduler might have scheduled a pod
-		// already when the kubelet does not know about its new taint yet. The
-		// kubelet will then refuse to launch the pod.
-		framework.ExpectNoError(e2epod.WaitForPodNotPending(cs, ns, tolerationPodName))
+		framework.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(cs, ns, tolerationPodName))
```

Comment:
This change should have been split out into a separate PR prior to promoting to Conformance, so we could verify whether the test still passed

Reply:
ack, sorry about that